clang/include/clang/Sema/Sema.h

```diff
@@ -1760,6 +1760,10 @@ class Sema final : public SemaBase {
   /// Add [[gsl::Pointer]] attributes for std:: types.
   void inferGslPointerAttribute(TypedefNameDecl *TD);
 
+  LifetimeCaptureByAttr *ParseLifetimeCaptureByAttr(const ParsedAttr &AL,
```

Comment:
nit: per LLVM code style, function name should start with a lower case letter, `openFile`

Is this function used in other place? I only see it is used in the implementation of `LazyProcessLifetimeCaptureByParams`, it is not necessary to be in the public header.

Reply:
You are right. Removed from header.

Reply:
Reverted. Looks like this was used by `static void HandleLifetimeCaptureByAttr(TypeProcessingState &State,` for parsing function type attribute.

Reply:
ah, I see. I missed that `HandleLifetimeCaptureByAttr` is an overload function.